packages/eas-cli/src/submissions/utils/config.ts

```diff
@@ -0,0 +1,26 @@
+import { ExpoConfig, getConfig } from '@expo/config';
+
+interface AppConfig {
+  owner?: string;
+  slug: string;
+}
+
+export function getAppConfig(projecDir: string): AppConfig {
+  const exp = getExpoConfig(projecDir);
+  if (!exp.slug) {
+    throw new Error('expo.slug is not defined in app.json');
+  }
+  return {
+    owner: exp.owner,
```

Comment:
```suggestion
    owner: exp.owner,
```
See the other comment about using `getProjectAccountNameAsync`